### ISSUE TITLE
A failing definite integral.

### ISSUE THREAD

**Issue by akhmerov** [sympy/sympy#2354]
I'm not sure if it's a known issue, since it lingers for already several versions. I have found the following example of a definite integral, that yields nonsense:

```
In [1]: import sympy

In [2]: sympy.__version__
Out[2]: '0.7.3'

In [3]: x = sympy.var('x')

In [4]: sympy.integrate(1/(2 + sympy.cos(x)), (x, 0, 2 * sympy.pi))
Out[4]: 0
```

As a side remark, if one puts a variable instead of 2 in the denominator, this becomes the integral with which I used to test whether it's time to abandon Mathematica and switch to sage/sympy.

**Comment by jrmuizel** [comment]
Running this today gives:
```
2*sqrt(3)*pi/3
```

which suggests this is fixed.

**Comment by normalhuman** [comment]
This is an exact duplicate of #13749 which was fixed by #13808 (and a test for it was added there).
